x/nexus/client/cli/query.go

```diff
@@ -192,16 +192,15 @@ func GetCommandTransferFee() *cobra.Command {
 
 			queryClient := types.NewQueryServiceClient(clientCtx)
 
-			amount, err := sdk.ParseCoinNormalized(args[2])
-			if err != nil {
+			if _, err := sdk.ParseCoinNormalized(args[2]); err != nil {
```

Comment:
Why not instantiate `TransferFeeRequest` directly and then use its new `GetAmount` function to validate? That way we avoid repeating code.

Reply:
Oh, yeah, forgot about that, thanks